prov/util/src/util_mem_monitor.c

```diff
@@ -99,6 +99,7 @@ void ofi_monitors_init(void)
 	cuda_monitor->init(cuda_monitor);
 	rocr_monitor->init(rocr_monitor);
 	ze_monitor->init(ze_monitor);
+	import_monitor->init(import_monitor);
```

Comment:
Is there always an `import_monitor`?

Reply:
Yes, there always is an import_monitor object, though the user doesn't have to import a monitor.

Reply:
```
struct ofi_import_monitor {                                                     
        struct ofi_mem_monitor monitor;                                         
        struct fid_mem_monitor *impfid;                                         
};                                                                              
                                                                                
static struct ofi_import_monitor impmon = {                                     
        .monitor.iface = FI_HMEM_SYSTEM,                                        
        .monitor.init = ofi_import_monitor_init,                                
        .monitor.cleanup = ofi_import_monitor_cleanup,                          
        .monitor.start = ofi_import_monitor_start,                              
        .monitor.stop = ofi_import_monitor_stop,                                
        .monitor.subscribe = ofi_import_monitor_subscribe,                      
        .monitor.unsubscribe = ofi_import_monitor_unsubscribe,                  
        .monitor.valid = ofi_import_monitor_valid,                              
};                                                                              
                                                                                
struct ofi_mem_monitor *import_monitor = &impmon.monitor;  
```

It's a static struct

Reply:
There should also be a matching cleanup call.